src/app/shared/utils/scripts-loader/external-script.model.ts

```diff
@@ -0,0 +1,21 @@
+export interface ExternalScript {
+  name: string;
+  src: string;
+}
+
+export enum ExternalScriptsNames {
+  ALTMETRIC = 'altmetric-embed',
+}
+
+export enum ExternalScriptsStatus {
+  LOADED = 'loaded',
+  ALREADY_LOADED = 'already loaded',
+  NOT_LOADED = 'not loaded',
+}
+
+export const ExternalScriptsList: ExternalScript[] = [
+  {
+    name: ExternalScriptsNames.ALTMETRIC,
+    src: 'https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js',
```

Comment:
I haven't had a chance to test this yet, but I wanted to quickly note that we'll need to make this URL configurable in some manner.  This sort of external CDN may not be compatible with GDPR.  It's also unclear to me if this is an "official" CDN for the altmetric badges or something else?

Basically, it'd unclear to me whether this would be GDPR compliant or not.   So, we need a way to change to configuration easily if sites need that.

Reply:
@sergius02 : Right, I understand this came from the official AltMetric documentation.  If you prefer to leave it as-is, we can do so.  But, I do think it's easier to maintain this code if the URL is configurable.  Consider this an *optional* request.

I also have realized that my feedback here is similar to the feedback in this [comment above](https://github.com/DSpace/dspace-angular/pull/2496#issuecomment-1887482111).   

The main point is that this external CDN (cloudfront.net) may not be GDPR compliant...but it doesn't appear there's any way to fix that issue since Altmetric requires you to use this CDN in order to use their badges.

So, in the end, we may need to simply add warnings on enabling Altmetric to the documentation since it *requires* usage of an external CDN (and we cannot make guarantees ourselves that the external CDN is GDPR compliant).